src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -152,19 +205,19 @@ private void ProcessDiscoveryFrame(Frame frame)
 
 					if (this.Log().IsEnabled(LogLevel.Debug))
 					{
-						this.Log().LogDebug($"Discovery: Loading {file}");
+						this.Log().LogDebug("Discovery: Loading {File}", file);
 					}
 
 					try
 					{
-						assemblies.Add(_loadContext.LoadFromAssemblyPath(file));
+						assemblies.Add(asmblyLoadContext.LoadFromAssemblyPath(file));
 					}
 					catch (Exception exc)
 					{
 						// With additional processors there may be duplicates of assemblies already loaded
 						if (this.Log().IsEnabled(LogLevel.Debug))
 						{
-							this.Log().LogDebug($"Failed to load assembly {file} : {exc}");
+							this.Log().LogDebug("Failed to load assembly {File} : {Exc}", file, exc);
```

Comment:
Did you want to use indexed formatting? Or is there something with the interpolation that does not work?

Reply:
One of the analyzers was complaining about using interpolation (I think because it's slower).

Reply:
I'm interested to know which one, it should be fine to use.

Reply:
https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca2254